goaviatrix/client.go

```diff
@@ -59,6 +60,9 @@ type Client struct {
 	ControllerIP     string
 	baseURL          string
 	IgnoreTagsConfig *IgnoreTagsConfig
+	cachedAccounts   []Account
```

Comment:
There might be extreme cases where a single terraform configuration has 2 configurations of the aviatrix provider.
Is this scenario completely unthinkable?
And if not, would this change have any impact on this?

Reply:
Nvm, it's per client, so it should of course have 2 caches for 2 controllers.